pkg/descheduler/apis/config/types_loadaware.go

```diff
@@ -64,6 +64,11 @@ type LowNodeLoadArgs struct {
 
 	// LowThresholds defines the low usage threshold of resources
 	LowThresholds ResourceThresholds
+
+	// AnomalyThresholds indicates the node load anomaly thresholds,
+	// the default is 5 consecutive times exceeding HighThresholds,
+	// it is determined that the node is abnormal, and the Pod needs to be migrated to reduce the load.
+	AnomalyThresholds *LoadAnomalyThresholds
```

Comment:
It seems to be a smooth threshold for node overutilization. Perhaps LoadHighThresholds is more concise?

Reply:
Already have a field named `HighThresholds`. Maybe named `LoadSmoothHighThresolds`? 😂

Reply:
Can we combine the fields of the high threshold? Or name it like `LoadHighConditions`...